### ISSUE TITLE
Failed test 'write content'

### ISSUE THREAD

**Issue by frithnanth** [pdf-raku/PDF-raku#19]
"zef install PDF" output:
```
===> Testing: PDF:ver<0.2.8>:auth<github:p6-pdf>:api<PDF-1.7>
# Failed test 'write content'
# at home#sources/26CC3C5D046C986175D965C1FD37A27CDA20209C (PDF::Grammar::Test) line 42
# expected: "[ (DOS\\r\\nCR1\\(\0) <444f530d0a435232> ] TJ"
#  matcher: 'json-eqv'
#      got: "[ (DOSrnCR1\\(\0) <444f530d0a435232> ] TJ"
# :ast({:content(${:TJ($[{:array($[{:literal("DOSrnCR1(\0")}, {:hex-string("DOS\r\nCR2")}])},])})})
# Looks like you failed 1 test of 21
===> Testing [FAIL]: PDF:ver<0.2.8>:auth<github:p6-pdf>:api<PDF-1.7>
```
Using Rakudo version 2018.03 on Linux

**Comment by dwarring** [comment]
@frithnanth . That looks as if it could be related to a JSON::Fast 0.9.9 bug https://github.com/timo/json_fast/issues/35. Fixed in JSON::Fast 0.9.10.

I'm reintroducing a work around, which should stop this test failing ob j::f 0.9.9

**Comment by frithnanth** [comment]
I'm using JSON::Fast:ver<0.9.11>, so I guess the bug is still lurking.
Anyway, your workaround works fine, thanks.
